test/CMakeLists.txt

```diff
@@ -223,6 +223,16 @@ if(MIOPEN_TEST_GFX1030)
     endif()
 endif()
 
+#TODO Code Quality WORKAROUND ROCm 5.1 update
+if(MIOPEN_BACKEND_OPENCL AND MIOPEN_TEST_ALL)
+    if(MIOPEN_TEST_GFX900 OR MIOPEN_TEST_GFX906)
+        list(APPEND SKIP_TESTS test_bn_3d_spatial_test test_conv3d test_immed_conv3d test_immed_conv2d)
+    endif()
+    if(MIOPEN_TEST_GFX1030)
+        list(APPEND SKIP_TESTS test_conv3d test_immed_conv3d test_immed_conv2d)
+    endif()
+endif()
```

Comment:
I highly recommend creating a ticket and mentioning the ticket number for this W/A here.